_config.yml

```diff
@@ -277,8 +277,11 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
-# youyan uid
-#youyan_uid: 
+# support youyan comments,
+# you can regist and get your uid from http://www.uyan.cc
+# for more informations, you can visit my blog https://jinfang134.github.io/posts/5f6c0475/
```

Comment:
No need to PR your blog in config file. All info about usage must be in conversatioan. For example, u can see my pulls.